doc/source/notebooks/understanding/models.ipynb

```diff
@@ -36,7 +36,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Create a very simple GPR model without building it in TensorFlow graph."
+    "Create a very simple GP regression model without building it in the TensorFlow graph:"
```

Comment:
maybe worth highlighting this a bit more - we *need* to wrap it in the `defer_build` because some of the things demonstrated in the following, such as adding transforms and priors, only works for *non*-compiled models, not necessarily obvious to people new to gpflow..